services/py-api/src/database/transaction_manager.py

```diff
@@ -141,7 +141,7 @@ async def callback(session):
             return result
 
         except Exception as e:
-            LOG.exception("Aborting transaction due to err {}".format(e))
+            LOG.warning("Aborting transaction due to err {}".format(e))
```

Comment:
Why? Aren't we losing the stack trace in this way? On line 133 we have a check for expected error. This means that the method we are calling is expected to return an err, which we have to handle. But what if some unexpected error arises via an unhandled exception? Well, we want to catch it as early as possible in this case, before it propagates up the stack trace. As this is an unexpected Exception, this means that we have a flaw in our code, and it would be nice to see where this flaw is coming from via the stack trace. As the exception is caught, the stack trace would not be displayed to the stderr unless we explicitly want it to, and we do want the stack trace for unexpected errors so that we could understand what happened. Otherwise here is what is going to happen. Some of the methods raisees an uncaught exception and it is cough here, and we just say "WARNING the transaction was aborted due to this err (and we show the final err in the stack trace, but we are missing the caused of it, the rest of the stack trace) https://docs.python.org/3/tutorial/errors.html#exception-chaining

This method returns an Err(Exception) and in the http_handler we have a check if the Err is not of concrete type for example Err(DuplikateEmailError) we return a 500 status code. And when we dig through the logs we see information, which is not helpful for properly tracing the error. I could be mistaken, but pls double check if this is what will happen.